handler.go

```diff
@@ -38,121 +39,121 @@ func (handler *Handler) LoggerMiddleware() gin.HandlerFunc {
 	}
 }
 
-//GetRouter configures all routes
-func (handler *Handler) GetRouter() http.Handler {
+// GetRouter configures all routes
+func (handler *Handler) GetRouter() *gin.Engine {
 	r := gin.New()
 	r.Use(handler.LoggerMiddleware())
 	gin.SetMode(gin.ReleaseMode)
 
 	r.POST("/major/:project", handler.OnMajor)
 	r.POST("/minor/:project", handler.OnMinor)
 	r.POST("/patch/:project", handler.OnPatch)
-	r.POST("/transient/minor/:version", handler.OnTransientMinor)
-	r.POST("/transient/patch/:version", handler.OnTransientPatch)
-	r.POST("/version/:project/:version", handler.OnSetVersion)
-	r.GET("/version/:project", handler.OnGetVersion)
+	r.POST("/transient/minor/:versionManager", handler.OnTransientMinor)
```

Comment:
i dont like the term "Manager" at all and especially inside a route

Reply:
That was not intended. Renaming variables in GoLand is broken. :(

Reply:
Commit d7804a fixes this.